Dockerfile.build-base

```diff
@@ -1,5 +1,5 @@
 #
-# This is the docker image used for ./bin/run-tests.sh and development tasks.
```

Comment:
Please delete this file if it is not used anymore by the ./bin/run-tests.sh script -- that was its only purpose.